charts/backbeat/templates/gc/deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "backbeat.fullname" . }}-gc-consumer
+  labels:
+    app: {{ template "backbeat.name" . }}-gc
+    chart: {{ template "backbeat.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.garbageCollector.consumer.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat.name" . }}-gc
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ template "backbeat.fullname" . }}-gc-consumer
```

Comment:
For the container name we can just use `consumer` or `gc-consumer` the entire backbeat name doesn't need to be templated in here as well since it exists under the `{{ template "backbeat.fullname" . }}-gc-consumer` pod. Otherwise LGTM

Reply:
ok, will change!